books/free-programming-books-langs.md

```diff
@@ -1205,6 +1205,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Full Stack JavaScript: Learn Backbone.js, Node.js and MongoDB](https://github.com/azat-co/fullstack-javascript) - Azat Mardan
 * [Introduction to Node.js](https://nodejs.dev/learn) (HTML)
 * [Mixu's Node Book](http://book.mixu.net/node/)
+* [Module Patterns](https://leanpub.com/modulepatterns) - Pedro Teixeira
```

Comment:
CONTRIBUTING guidelines needs to be enforced. Be sure next time to satisfy them :wink: 
- resource formats
- leanpub notes

```suggestion
* [Module Patterns](https://leanpub.com/modulepatterns) - Pedro Teixeira (HTML, PDF, EPUB, Kindle) *(Leanpub account or valid email requested)*
```